### DISCUSSION TITLE
Bounding Box Coordinates Outside the Image

### DISCUSSION THREAD

**Discussion by Pluto028** [ultralytics/ultralytics#15931] [Q&A] [answered]
Hello! Please help me as a beginner.

Situation:
I have an image with multiple elements enclosed within bounding boxes.
These bounding boxes are specified by coordinates.
I need to split this image for training purposes.
I’ve encountered a situation where the coordinates specified fall outside the bounds of the split image.

Questions:
What happens if coordinates are specified outside the bounds of the split image?
Is there any difference in the outcome compared to when the coordinates are aligned with the edge of the image?

**Comment by glenn-jocher** [comment, marked as answer]
@Pluto028 hello! If bounding box coordinates fall outside the bounds of the split image, they may be ignored or cause errors during training. It's best to ensure that all coordinates are within the image boundaries. You can adjust or clip the coordinates to fit within the image dimensions.

**Reply by Pluto028** [reply]
Thank you for the information!

**Reply by glenn-jocher** [reply]
@Pluto028 you're welcome! If you have any more questions or need further assistance, feel free to ask.
